app/schemas/data.py

```diff
@@ -0,0 +1,56 @@
+from typing import Optional
+
+from pydantic import BaseModel, ConfigDict
+import datetime
+
+
+class CreateData(BaseModel):
+    model_config = ConfigDict(from_attributes=True)
+
+    date: datetime.date
+    time: datetime.time
+    nuts3: str
+    nuts2: str
+    temperature_air: Optional[float]
+    relative_humidity: Optional[float]
+    precipitation: Optional[float]
+    wind_speed: Optional[float]
+    wind_direction: Optional[float]
+    wind_gust: Optional[float]
+    atmospheric_pressure: Optional[float]
+    relative_humidity_canopy: Optional[float]
+    temperature_canopy: Optional[float]
+    solar_irradiance_copernicus: Optional[float]
+
+class UpdateData(BaseModel):
+    model_config = ConfigDict(from_attributes=True)
+
+    nuts3: Optional[str] = None
```

Comment:
why do you add default value `None` ?

Reply:
I'm not quite sure, I removed them because the values should not be None if not passed, even though at the moment the crud isn't being used anywhere